src/examples/forms/multithumbrange/MultiThumbRangeHint.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { Field, Label, Hint, MultiThumbRange } from '@zendeskgarden/react-forms';
+
+const Example = () => {
+  const [minVal, setMinVal] = useState<number | undefined>(20);
+  const [maxVal, setMaxVal] = useState<number | undefined>(80);
```

Comment:
As @hzhu mentioned, I think you can take advantage of type inference here with `useState(20)` and `useState(80)`

Reply:
Whoops, I think `minValue` and `maxValue` from the multi-thumb range component's `onChange` returns these values typed as `number | undefined`. So I think typing the state out seems okay (otherwise it would need to be type casted when the state is yet).